docker/base.Dockerfile

```diff
@@ -76,7 +76,7 @@ RUN if [ "$ZENML_NIGHTLY" = "true" ]; then \
       PACKAGE_NAME="zenml"; \
     fi \
     && pip install --upgrade pip \
-    && pip install "${PACKAGE_NAME}[server,secrets-aws,secrets-gcp,secrets-azure,secrets-hashicorp,s3fs,gcsfs,adlfs,connectors-aws,connectors-gcp,connectors-azure]${ZENML_VERSION:+==$ZENML_VERSION}" \
+    && pip install "${PACKAGE_NAME}[server,secrets-aws,secrets-gcp,secrets-azure,secrets-hashicorp,s3fs,gcsfs,adlfs,connectors-aws,connectors-gcp,connectors-azure,azureml,sagemaker,vertex]${ZENML_VERSION:+==$ZENML_VERSION}" \
```

Comment:
Why do you need to install these in the client image ? You already get them when users install integrations required for the orchestrator in their stack, right ? Installing these for everyone is not only unnecessary but also makes the images bigger (something are users are already complaining about).

Reply:
This is for building the server image with the `server-builder`, isn't it? Maybe I am missing something here.

Reply:
You're right, I misread this, I apologize.